frontend/openchat-client/src/openchat.ts

```diff
@@ -4592,44 +4546,32 @@ export class OpenChat extends OpenChatAgentWorker {
                 }
                 return false;
             })
-            .catch((err) => {
-                this._logger.error("Unable to unfreeze group", err);
-                return false;
-            });
+            .catch(() => false);
     }
 
     deleteFrozenGroup(chatId: GroupChatIdentifier): Promise<boolean> {
         return this.sendRequest({ kind: "deleteFrozenGroup", chatId })
             .then((resp) => resp === "success")
-            .catch((err) => {
-                this._logger.error("Unable to unfreeze group", err);
-                return false;
-            });
+            .catch(() => false);
     }
 
     addHotGroupExclusion(chatId: GroupChatIdentifier): Promise<boolean> {
         return this.sendRequest({ kind: "addHotGroupExclusion", chatId })
             .then((resp) => resp === "success")
-            .catch((err) => {
-                this._logger.error("Unable to add hot group exclusion", err);
-                return false;
-            });
+            .catch(() => false);
     }
 
     removeHotGroupExclusion(chatId: GroupChatIdentifier): Promise<boolean> {
         return this.sendRequest({ kind: "removeHotGroupExclusion", chatId })
             .then((resp) => resp === "success")
-            .catch((err) => {
-                this._logger.error("Unable to remove hot group exclusion", err);
-                return false;
-            });
+            .catch(() => false);
     }
 
     addMessageFilter(regex: string): Promise<boolean> {
         try {
             new RegExp(regex);
         } catch (e) {
-            this._logger.error("Unable to add message filter - invalid regex", regex);
+            console.error("Unable to add message filter - invalid regex", regex);
```

Comment:
this is not a response to a call to `sendRequest`. I changed this because it doesn't seem worth logging (to rollbar) as there is no real action we could take about it, it's just representative of user input error.

Reply:
Ahh right yup